### DISCUSSION TITLE
[Feature] Upload folder watches for new files/import folder

### DISCUSSION THREAD

**Discussion by KWall24** [immich-app/immich#7515] [Feature Request]
### The feature

I'd like to see a way to have Immich periodically look for files in a folder that it can import into Immich, either from the /library/upload folder or another folder.

Example:
I have a folder "summer" containing 100 different jpg files
I copy that folder using my computer's file manager (or however I want) into /immich/library/upload, giving me /immich/library/upload/summer
Immich checks the /upload folder for new content on a schedule.
Immich sees the folder "summer" has been added and begins to import these files, treating them like any other import/upload - applying storage templates if enabled, and if not, moving into the correct folders as though they were uploaded from the webui
Once a file is imported, it deletes the file from the "summer" folder.
Once the folder "summer" is empty, it deletes it

This would be useful for users who are bringing in content from various places/devices and would like it organized in a consistent fashion.

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
Duplicate of  #1699. You can also do this using the file watcher for external libraries.

**Reply by HansAndreManfredson** [reply]
But without the ability to deleting files ;-)
👎
